File: Day302/minimizeMaximumPairSumInArray.java
class Solution {
    public int minPairSum(int[] nums) {
        Arrays.sort(nums);
        int answer = 0;
        int i =0, j= nums.length-1;
        while (i<j) {
            answer = Math.max(answer, nums[i++]+nums[j--]);
        }   
        return answer;
    }
}

File: Day302/minimizeMaximumPairSumInArray.go
func minPairSum(nums []int) int {
    sort.Ints(nums)
    i, j := 0, len(nums) -1 
    answer := 0
    for i<j {
        answer = int(math.Max(float64(nums[i] + nums[j]),  float64(answer)))
        i++
        j--
    }
    return answer
}